tests/e2e/test/e2e/oc_mirror_v2.go

```
@@ -242,20 +239,11 @@ var _ = g.Describe("[OTP][sig-cli] Workloads ocmirror v2 works well", func() {
-				e2e.Logf("copy failed, retrying...")
-				return false, nil
-			}
-			return true, nil
-		})
-		compat_otp.AssertWaitPollNoErr(waitErr, fmt.Sprintf("max time reached but the skopeo copy still failed"))
+		skopeExecute(fmt.Sprintf("skopeo copy --all docker://registry.redhat.io/redhat/redhat-operator-index:v4.16 oci://%s  --remove-signatures --insecure-policy --authfile %s", dirname+"/redhat-operator-index", dirname+"/.dockerconfigjson"))
 
 		compat_otp.By("Start mirror2mirror")
 		defer os.RemoveAll(".oc-mirror.log")
-		waitErr = wait.PollImmediate(300*time.Second, 3600*time.Second, func() (bool, error) {
+		waitErr := wait.PollImmediate(300*time.Second, 3600*time.Second, func() (bool, error) {
```

golangci-lint (2.13.2)

[error] SA1019: k8s.io/apimachinery/pkg/util/wait.PollImmediate is deprecated: This method does not return errors from context, use PollUntilContextTimeout. Note that the new method will no longer return ErrWaitTimeout and instead return errors defined by the context package. Will be removed in a future release. 

(staticcheck)